README.md

```diff
@@ -21,33 +21,33 @@ Click on the name of a plugin or module to view that content's documentation:
     - [k8s](https://docs.ansible.com/ansible/2.10/collections/community/kubernetes/k8s_lookup.html)
   - **Modules**:
     - [k8s](https://docs.ansible.com/ansible/2.10/collections/community/kubernetes/k8s_module.html)
```

Comment:
Is the documentation for these modules not present in this repo/collection?

Reply:
Do you mean are there generated docs in this repo? If so, then no, there aren't. My counter is how do I do that? Because I would like to.

Reply:
https://github.com/ansible-network/collection_prep/blob/master/collection_prep/cmd/add_docs.py should generate rst files into /docs/ from the module DOCUMENTATION block.  For the AWS collections I run that as part of a release prep PR, ex: https://github.com/ansible-collections/amazon.aws/pull/254

Reply:
@jillr @gravesm Let us handle this in a separate PR.